frontend/assets/javascripts/src/utils/form/Password.js

```diff
@@ -79,9 +70,9 @@ function ($, bean, component) {
 
         require(['js!zxcvbn'], function() {
 
-            var $passwordStrengthIndicatorTemplate = self.$passwordStrengthIndicatorTemplate();
-
-            $passwordStrengthIndicatorTemplate.insertAfter(self.getElem('PASSWORD_STRENGTH_INPUT'));
+            utilsHelper.toArray(document.querySelectorAll('.' + self.classes.STRENGTH_INDICATOR)).map(function (elem) {
+                elem.classList.toggle('is-hidden');
+            });
```

Comment:
Yup agreed, been speaking with @davidrapson about a gradual move to this was along with an aligning of how we do our JavaScript - psuedo constants, removing componant.js, changing un-needed constructor modules to object literals etc. 
But yes why not we could do this as one big task. Perhaps a little get together to discus this approach and the best way forward at some point?